src/components/custom/_dtrading.js

```diff
@@ -0,0 +1,104 @@
+import React from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import device from 'themes/device'
+import { Container, SectionContainer } from 'components/containers'
+import { Header, Text, Image } from 'components/elements'
+
+const StyledSection = styled(SectionContainer)`
+    background-color: var(--color-white);
+    box-shadow: inset 1px 0 0 1px var(--color-grey-2);
+
+    @media ${device.tabletL} {
+        padding: 1.74rem 0 4rem 0;
+    }
+`
+const StyledContainer = styled(Container)`
+    display: flex;
+    flex-direction: column;
+
+    div:first-child {
+        margin-top: 0;
+    }
+`
+const Content = styled.div`
+    width: 100%;
+    max-width: 58.8rem;
+    display: flex;
+    flex-direction: column;
+    margin-right: ${props => props.margin_right};
+
+    ${Text} {
+        margin-top: 0.8rem;
+    }
+
+    @media ${device.tabletL} {
+        margin: 0 auto;
+    }
+`
+
+const ImageWrapper = styled.div`
+    max-width: 58.8rem;
+    width: 100%;
+    max-height: 30rem;
+    margin-right: ${props => props.margin_right};
+
+    @media ${device.tabletL} {
+        margin: 2rem auto;
+    }
+`
+const StyledHeader = styled(Header)`
+    margin-top: ${props => props.second_title_margin ? '2.4rem' : '4rem'};
+    font-size: 3.6rem;
+    line-height: 1.25;
+
+    @media ${device.tabletL} {
+        margin-top: 0;
+    }
+`
+const Row = styled.div`
+    flex-direction: ${props => props.flex_direction};
+    width: 100%;
+    display: flex;
+    margin-top: 4rem;
+
+    @media ${device.tabletL} {
+        flex-direction: column;
+    }
+`
+const DTrading = ({ trading, reverse, two_title }) => {
+    return (
+        <StyledSection>
+            <StyledContainer>
+                {trading.map((item, index) => {
+                    let is_even = reverse ? (index + 1) % 2 === 0 : index % 2 === 0
```

Comment:
why do we need to pass in reverse props?
everything modulus 2 will be either 1 or 0, so you can use `index % 2` only no need to do `index % 2 === 0`

Reply:
done